targo_test.go

```diff
@@ -32,6 +32,20 @@ var (
 	voidPath          = parentPath + "/void"
 )
 
+//A path without an ending slash will produce a tar
```

Comment:
I wanted to try the [Godoc example mechanism](https://blog.golang.org/examples), you don't like it ? I tried to open the documentation of targo and the examples correctly shows under the document of the Create function. 

I also think that it would be nicer to add something about the output but I have trouble to find something nice to do.